Homework/homework6/test_task2.py

```diff
@@ -0,0 +1,32 @@
+from homework6.task2 import instances_counter
+
+
+def test_instances_are_counted():
+    @instances_counter
+    class Cls:
+        pass
+
+    Cls()
+
+    assert Cls.instances_counter == 1
```

Comment:
Which additional value this tests add to the `test_get_created_instances`?

Reply:
It was a logic behind that if the variable changes right here but test `test_get_created_instances` fails than smth wrong with the function... But okay, I see, maybe such situation is not likely for such function.